website/source/community.html.erb

```diff
@@ -27,6 +27,12 @@ active, dedicated users willing to help you through various mediums.
 	on GitHub</a>. Please only use this for reporting bugs. Do not ask
 for general help here. Use IRC or the mailing list for that.
 
+<h1>Community Tools:</h1>
+<p>These Nomad projects are created and managed by the amazing members of the Nomad community:</p>
+<ul>
+	<li><a href="https://github.com/iverberk/nomad-ui">nomad-ui</a> - User Interface in a single docker container</li>
```

Comment:
Change description to:

"Nomad UI is a simple to deploy, web based UI for interacting with Nomad"